docroot/.htaccess

```diff
@@ -69,6 +69,10 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
+  # Forbid access to admin and node paths on the api2.foia.gov domain.
+  RewriteCond %{HTTP_HOST} api2.foia.gov
```

Comment:
After thinking this through a bit more, instead of specifying which domains we want to block, maybe we should block all domains that aren't some variation of `admin.foia.gov`:

```
RewriteCond %{HTTP_HOST} !(.*)admin\.foia\.gov [NC]
```

The NC makes it case-insensitive.